docs/integrations/development.md

```diff
@@ -1526,3 +1526,63 @@ immediately reflect that in the graph.
 We hope to provide a good interface via this SDK for providing an interface for
 handling events via this SDK and providing commands/utilities for testing events
 to get an understanding of how they may affect the JupiterOne graph.
+
+## j1 CLI
+
+In addition to the `j1-integration` CLI, this project exposes a `j1` CLI for
+`import` and `export` processes.
+
+### Authentication
+
+For commands that require interaction with JupiterOne's API, the CLI will
+provide ways of inputing credentials. To support that, all commands that
+interact with an API will accept an `--api-key` option.
+
+For convenience when developing locally, we will also look for a
+`JUPITERONE_API_KEY` environment variable for an API key to use.
+
+### Supported commands
+
+Currently, the CLI supports a limited interface consisting of only two commands:
+`export` and `import`.
+
```

Comment:
Might be nice to include the required options?

Reply:
Required is a bit odd for these, since so many of them either have a default or will use .env values if provided that way.  

I think the only option truly flagged as a required one is the scope for the import command.  I can get that added in a later PR